iothub/service/src/FeedbackRecord.cs

```diff
@@ -7,25 +7,51 @@ namespace Microsoft.Azure.Devices
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Contains properties of an individual feedback record. 
+    /// Contains properties of an individual feedback record.
     /// </summary>
     public class FeedbackRecord
     {
+        /// <summary>
+        /// The message Id of the cloud-to-device message to which this feedback information relates.
+        /// </summary>
         [JsonProperty(PropertyName = "originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string OriginalMessageId { get; set; }
 
+        /// <summary>
+        /// String values for StatusCode <see cref="FeedbackStatusCode"/>.
```

Comment:
This will read funny when generated because the link will not make sense in the preceding sentence.

Reply:
Should I reword to - String values for <see cref="FeedbackStatusCode"/> ?
or
String values for FeedbackStatusCode <see cref="FeedbackStatusCode"/> ?

Reply:
yes. I have not pushed it in yet as you were still reviewing.